pages/care-package/[guid]/broker-care-charges.js

```diff
@@ -157,38 +190,68 @@ const CareCharge = () => {
   const isOngoing = watch('isOngoing');
   const cost = watch('cost');
 
-  const isLoading = coreLoading || calculatedCostLoading || careChargeLoading || isRequestBeingSent;
-  const isS117Client = packageInfo?.settings?.isS117Client;
+  const isS117Client = coreInfo?.settings?.isS117Client;
+
+  const buttonProps = useMemo(() => {
```

Comment:
this is hard to read, change it back to how it was. no need to over engineer a solution